### ISSUE TITLE
Object Layer misaligned with Tile Layers

### ISSUE THREAD

**Issue by JoleMile** [lafriks/go-tiled#42]
Hello,
first time Golang and game dev here so this is more of a question than a bug report :) 

I'm facing an issue where `image.NRGBA` produced by `rend.RenderVisibleLayers()` is not aligned with objects from the `tiled.ObjectGroup`. They seem flipped vertically. I am using the `pixel` 2d library for rendering.

This is what my `Tiled` editor looks like:
![Screen Shot 2020-12-05 at 10 28 43 AM](https://user-images.githubusercontent.com/42361585/101216609-ae523a80-36e4-11eb-8b36-16d8a1463b4b.png)

And this is The result I get after rendering everything without moving or scaling things:
![Screen Shot 2020-12-05 at 10 31 21 AM](https://user-images.githubusercontent.com/42361585/101216811-0be68700-36e5-11eb-9ac6-f40684f91a7b.png)

Purple lines indicate the (0, 0) point.
Red boxes are representing where the `ObjectGroup` objects are.

Why are objects flipped compared to the rendered map and the compared to how they look in the editor?
Is this the expected behaviour or am I doing something wrong?
Any help would be greatly appreciated :)

**Comment by Qwarkster** [comment]
The culprit is in that tiled orients at the top left, and pixel orients from the bottom left. I encountered a similar issue, I believe my solution was when loading the objects from the object group, I inverted the Y pos, by adjusting the Y pos to be the max Y position minus the file Y position.

`newY := newMap.Size.Y - obj.Y - obj.Height`

That's the line right out of my own code that gets the proper Y position to use when rendering on screen. Depending on how you load, it might vary.
